ppr-ui/src/composables/useParty.ts

```diff
@@ -30,6 +30,9 @@ export const useParty = () => {
       party.address.region +
       ' ' +
       party.address.postalCode
+    if (party.address.country) {
+      address = address + '<br>' + party.address.country
```

Comment:
does this need a </br> at the end?

Reply:
Don't think so... it already has cell padding